youtube_dl/downloader/__init__.py

```diff
@@ -30,14 +31,17 @@ def get_suitable_downloader(info_dict, params={}):
     protocol = determine_protocol(info_dict)
     info_dict['protocol'] = protocol
 
+    if (info_dict.get('start_time') or info_dict.get('end_time')) and FFmpegFD.available() and FFmpegFD.supports(info_dict):
```

Comment:
Some convenience method like `can_download(info_dict)` checking for `available() and supports(info_dict)` would be good (this is used twice in the code).